frontend/src/__tests__/cypress/cypress/utils/mockUsers.ts

```diff
@@ -108,38 +108,86 @@ const setUserConfig = (userConfig: UserConfig = {}, isAllowed = true) => {
     },
   );
 
-  const PROJECT_ADMIN_RESOURCES = ['projects', 'rolebindings'];
-  const EDIT_VERBS = ['*', 'create', 'delete', 'deletecollection', 'update', 'patch'];
   cy.interceptK8s('POST', SelfSubjectAccessReviewModel, (req) => {
-    const { verb, group, resource, namespace } = req.body.spec.resourceAttributes;
+    const { resourceAttributes } = req.body.spec;
     req.reply(
       mockSelfSubjectAccessReview({
-        verb,
-        group,
-        resource,
-        namespace,
-        allowed:
-          // cluster admin can do everything
-          isClusterAdmin
-            ? true
-            : // self provisioner capabilities grant access to project creation
-            resource === 'projectrequests'
-            ? isSelfProvisioner
-            : // project admins
-            PROJECT_ADMIN_RESOURCES.includes(resource) && EDIT_VERBS.includes(verb)
-            ? isProjectAdmin
-            : isProductAdmin
-            ? // product admins are getting direct access to resources in the deployment namespace (but importantly, not other projects)
-              namespace === 'opendatahub'
-              ? true
-              : // product admins will be limited to listing resources
-                !EDIT_VERBS.includes(verb)
-            : // everyone else can perform any action within
-              !!namespace,
+        ...resourceAttributes,
+        allowed: evaluateAccess(resourceAttributes, {
+          isClusterAdmin,
+          isProductAdmin,
+          isProjectAdmin,
+          isSelfProvisioner,
+        }),
       }),
     );
   });
 
+  const evaluateAccess = (
+    resourceAttributes: {
+      verb: string;
+      group?: string;
+      resource: string;
+      namespace?: string;
+    },
+    config: UserConfig,
+  ): boolean => {
+    const { verb, resource, namespace } = resourceAttributes;
+    const {
+      isClusterAdmin: clusterAdmin,
+      isProductAdmin: productAdmin,
+      isProjectAdmin: projectAdmin,
+      isSelfProvisioner: selfProvisioner,
+    } = config;
+
+    const PROJECT_ADMIN_RESOURCES = ['projects', 'rolebindings'];
+    const EDIT_VERBS = ['*', 'create', 'delete', 'deletecollection', 'update', 'patch'];
+
+    if (clusterAdmin) {
+      Cypress.log({ message: 'Cluster admin can do everything' });
+      return true;
+    }
+
+    if (resource === 'projectrequests' && selfProvisioner) {
+      Cypress.log({ message: 'Self provisioner capabilities grant access to project creation' });
+      return true;
+    }
+
+    if (resource === 'rolebindings' && !projectAdmin) {
+      Cypress.log({ message: 'Users cannot access permissions tab' });
+      return false;
+    }
+
+    if (PROJECT_ADMIN_RESOURCES.includes(resource) && EDIT_VERBS.includes(verb)) {
```

Comment:
Project admin checks should be UNDER Product admins

* Cluster-Admin
* Product Admin
* Project Admin
* Regular project user (today we only support edit -- but edit then view would be how I expect that)
* fallback checks